msteams-platform/sbs-file-handling-in-bot.yml

```diff
@@ -0,0 +1,327 @@
+        | Scopes | Enter **User.Read** and also add all the **Scopes** required while specifying permissions to downstream APIs. |
+
+        :::image type="content" source="./assets/images/teams-file-upload-bot/oauth-settings.png" alt-text="Bot service connection":::
+
+    1. Select **Save**.
+
+- title: Test your app
+  durationInMinutes: 2
+  content: |
+    The most comprehensive way to test your app is by creating an app package and uploading it to Teams. This method is the only way to test the full functionality available to your app,
+    across all scopes. You can configure and run the Teams app by uploading the manifest.zip file.
+
+    **To run the Teams file bot, you need to upload the manifest.zip file in Teams. Perform the following steps:**
+
+    1. In the **File Explorer**, go to **botbuilder-samples** > **samples** > **csharp_dotnetcore** > **56.teams-file-upload** > **appsettings**.
+
+    1. Open the appsettings in visual studio and update the appsettings.json configuration for the bot to use the `MicrosoftAppId (Bot Id)`, `MicrosoftAppPassword(Value of the client secret ID)`, `MicrosoftAppType`, `MicrosoftAppTenantID`
+
+       ![app settings](~/assets/images/teams-file-upload-bot/app-settings.png)
+    
+    1. Open the **manifest.json** file in visual studio.
+
+    1. In the manifest.json file, find YOUR-MICROSOFT-APP-ID and replace it with your App Id (that was created when you registered your bot earlier) everywhere you see the place holder string and save it.
+       
+       ![manifest json](~/assets/images/teams-file-upload-bot/manifest-json.png)
+
+    1. Zip up the contents of the `TeamsAppManifest' folder to create **manifest.zip**. Ensure to open the manifest file in visual studio code and replace with your MicrosoftAppID (depending on the scenario the Microsoft App Id may occur multiple times in the manifest.json)
+
+    1. Upload the manifest.zip to Teams (in the Apps view click "Upload a custom app")
+
+    1. To upload **manifest.zip**, select **Store > Manage your apps > Upload a custom app** and open **manifest.zip**.
+
+    1. Select **Add** to add the bot to your chat.
+
+    1. Before sending a message to the bot, go to **File Explorer**, navigate to **botbuilder-samples** > **samples** > **csharp_dotnetcore** > **56.teams-file-upload**
+
+    1. Open **TeamsFileUpload.csproj** in visual studio and ensure you are in 'Teams file upload'
+
+        ![Visual code](~/assets/images/teams-file-upload-bot/visual-studio-teams-file-upload-bot.png)
+
+    1. Run your bot, from Visual Studio with `F5` or using `dotnet run` in the appropriate folder
+
+    1. Send a message `Hi` to the bot and wait for few seconds for it to respond with a card.
+
+- title: Interacting with the bot in Teams
+  durationInMinutes: 2
+  content: |
+    > [!NOTE]
+    >  The bot is installed in "personal" scope which is why you immediately enter a one on one chat conversation with the bot. See Teams documentation for more details.
```

Comment:
If we suggest to see some docs, link to those.

@v-silakshmi

Reply:
made changes to the document.